geometry/bounding_box.cpp

```diff
@@ -24,4 +24,9 @@ bool BoundingBox::HasPoint(double x, double y) const
 {
   return x >= m_min.x && x <= m_max.x && y >= m_min.y && y <= m_max.y;
```

Comment:
`return HasPoint(x, y, 0.0);` ?

Reply:
Functionally your proposal is correct but slightly less efficient, therefore I'd prefer to keep this as is.

Reply:
Ok